extensions/security/runtime-spi/src/main/java/io/quarkus/security/spi/runtime/DisabledAuthorizationController.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkus.security.spi.runtime;
+
+import javax.annotation.Priority;
+import javax.enterprise.inject.Alternative;
+import javax.inject.Singleton;
+import javax.interceptor.Interceptor;
+
+/**
+ * Controller used in dev mode if {@code quarkus.security.authorization.enabled=false}.
+ */
+@Alternative
+@Priority(Interceptor.Priority.LIBRARY_AFTER)
+@Singleton
+public class DisabledAuthorizationController extends AuthorizationController {
+
+    public boolean isAuthorizationEnabled() {
+        return false;
```

Comment:
Call me paranoid, but can you add a LaunchMode.getCurrent() check here to validate that it is devmode, and throw an exception otherwise? Also maybe change the class name to indicate that it is dev mode only.